scripts/clarity-icons-svg.js

```diff
@@ -12,7 +12,8 @@ const SHAPE_SETS = [
 ];
 
 writeSVGIcons(SHAPE_SETS, () => {
+    shell.exec("cd dist/clarity-icons/shapes; zip -r all-shapes.zip ./**/*");
     SHAPE_SETS.forEach((setName) => {
-        shell.exec(`zip -r dist/clarity-icons/shapes/svg-source/${setName}.zip dist/clarity-icons/shapes/svg-source/${setName}`);
```

Comment:
So we getting rid of the `svg-source` folder altogether?

Reply:
correct. that wasn't there in the earlier version. meant to be a 'holding place' that gulp task used to use but it's not in the npm package.